src/main/drivers/rangefinder_hcsr04.h

```diff
@@ -19,4 +19,6 @@
 
 #include "drivers/rangefinder.h"
 
+#define RANGEFINDER_HCSR04_TASK_PERIOD_MS 50
```

Comment:
@martinbudden this probably should be determined from the scratch. See this:

https://github.com/iNavFlight/inav/blame/master/src/main/fc/fc_tasks.c#L456

50/70/20